app/service.js

```diff
@@ -48,6 +51,12 @@ app.post('/v1/metadata', function(req, res) {
     return;
   }
 
+  const maxUrls = app.get('maxUrls');
+  if (req.body.urls.length > maxUrls) {
+    fail(errorMessages.maxUrls, 400);
```

Comment:
I'm a bit confused on the desired behavior.
If I send you 35 URLs (with a max of 20), should it:

a) Fail and give me nothing?
b) Return me the first 20 results and ignore the rest?
c) Return me the first 20 results, and return an error response for the rest?

For example,

```js
  {
    request_error: "",
    url_errors: {
      "http://www.mozilla.org/21": "Fetch Error: maxUrls exceeded"
    },
    urls: {
      "http://www.mozilla.org/1": {},
      ...
      "http://www.mozilla.org/20": {}
    }
  }
```

/cc @k88hudson